src/long.js

```diff
@@ -1,4 +1,6 @@
 module.exports = Long;
+module.exports.default = Long;
+module.exports.Long = Long;
```

Comment:
By adding this line someone can `import { Long } from "long";`. This also paves the way to export multiple things from the module for future.

In node.js land users could require the package as follows:
```js
const { Long } from "long";
```